source/Projectile.js

```diff
@@ -0,0 +1,23 @@
+var Pixi = require("pixi.js")
+var Reference = require("Reference.js")
+
+export default class Projectile extends Pixi.Sprite {
+    constructor(x, y) {
+        var projectileTex =
+            Pixi.Texture.fromImage(require("./images/projectile.png"))
```

Comment:
Hmm... We've got a fairly small number of textures in Starjunk so a texture library may be a bit overkill. My inclination would be to go with the former and move the texture loading out of the class, but not go whole hog and give us another file.

Anyone else wanna weigh in on this with a preference?